docs/connecting_to_remote_db_aws_aurora.md

```diff
@@ -0,0 +1,112 @@
+# Connecting to a remote Klaxon database in an AWS Aurora cluster
+
+This documentation assumes several things:
+- The app is running locally in a Docker container
+- You have already created an Aurora RDS cluster where the Klaxon database will be housed. That cluster should be configured in a way that is compatible with PostgreSQL 14. For our purposes at the Post, we used `database_engine_version: 14.4` and `instance_size: db.t4g.medium`. [Here is a guide](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Support) to help you select a compatible instance size. We deploy our AWS resources in infrastructure-specific repos, however resources [can also be created in the AWS console](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Aurora.CreateInstance.html).
+- You have master credentials to access the Postgres database.
+
+## Create the database and app user
```

Comment:
am i right that this section and the one subsequent shouldn't ever be necessary except the first time someone on our team sets up this project? if so i'd note that here